changes/ticket33072

```diff
@@ -0,0 +1,3 @@
+  o Minor bugfix (directory authority):
+    - Always allow compressed directory requests. Fixes bug 33072; bugfix on
```

Comment:
Oh, the changes file is out of sync as well.

We let AuthDirRejectRequestsUnderLoad and the bandwidth buckets decide about compressed requests.